lbrynet/daemon/Daemon.py

```diff
@@ -2326,39 +2326,44 @@ def jsonrpc_publish(self, name, bid, metadata=None, file_path=None, fee=None, ti
 
     @requires(WALLET_COMPONENT, conditions=[WALLET_IS_UNLOCKED])
     @defer.inlineCallbacks
-    def jsonrpc_claim_abandon(self, claim_id=None, txid=None, nout=None):
+    def jsonrpc_claim_abandon(self, claim_id=None, txid=None, nout=None, account_id=None):
         """
         Abandon a name and reclaim credits from the claim
 
         Usage:
             claim_abandon [<claim_id> | --claim_id=<claim_id>]
                           [<txid> | --txid=<txid>] [<nout> | --nout=<nout>]
+                          [--account_id=<account_id>]
 
         Options:
             --claim_id=<claim_id> : (str) claim_id of the claim to abandon
             --txid=<txid> : (str) txid of the claim to abandon
             --nout=<nout> : (int) nout of the claim to abandon
+            --account_id=<account_id> : (str) id of the account to use
```

Comment:
I would like to standardize on using `--account=` across all commands. I'll update the other commands today. Function argument itself should remain `<account_id>`.

Reply:
Same as above